packages/web-app-files/src/components/SideBar/Details/FileDetailsMultiple.vue

```diff
@@ -16,6 +16,10 @@
           <th scope="col" class="oc-pr-s" v-text="foldersText" />
           <td v-text="foldersCount" />
         </tr>
+        <tr data-testid="spacesCount">
```

Comment:
Please add a prop to the panel to only show the spaces count on "meta-pages". In a subfolder of a space (personal / project / whatever) it doesn't make much sense to show `Files: x, Folders: y, Spaces: 0` and might confuse users.

This works by defining a prop on the panel and then adding a getter for the prop name in an `componentAttrs` object in `fileSideBars.ts` of the files app.